supplementary_style_guide/glossary_terms_conventions/general_conventions/b.adoc

```diff
@@ -174,13 +174,15 @@ The practice of having both modes together is often referred to as "hybrid", "ag
 [discrete]
 [[bios]]
 ==== image:images/yes.png[yes] BIOS (noun)
-*Description*: "BIOS" is an abbreviation for basic input and output system. The plural form is "BIOSes".
+*Description*: "BIOS" is an abbreviation for Basic Input/Output System. The plural form is "BIOSes". BIOS is the specific name for the motherboard firmware that provides runtime services for operating systems in older PCs. Modern computers use a different kind of firmware, called either EFI or UEFI. 
 
-*Use it*: yes
+Do not use _BIOS_ as a generic term to refer to computer firmware. Use _firmware_ or a specific phrase like "UEFI firmware" or "legacy BIOS" instead.
```

Comment:
Maybe also expand EFI and UEFI on first occurrence?

Reply:
Noted on the IBM style rec -- thanks!
Now I'm wondering if introducing EFI and UEFI requires entries or guidance on whether to expand the acronyms if you're using them (in non-SSG content). Any thoughts?

Reply:
I would just treat them as normal (not well known) abbreviations, and expand them on first occurrence.

Reply:
Like BIOS, EFI and UEFI are hardly ever used in expanded forms that I have encountered. That being said, it might be a good idea to create separate entries for them in a separate PR, but I would argue against expanding them here.

Reply:
Fair enough -- I'm not familiar with the area so I'll defer to Brian's expertise :)